src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector128.cs

```diff
@@ -752,6 +762,26 @@ public static unsafe Vector128<byte> Create(byte e0, byte e1, byte e2, byte e3,
                 return Sse2.UnpackLow(lo64, hi64).AsByte();                                         // <  0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12, 13, 14, 15 >
             }
 
+            if (Arm.AdvSimd.IsSupported)
```

Comment:
Why does `ulong` but not `long` need the fallback?

Reply:
Nevermind, I misread the diff. This is on `Vector128<byte>`. I apparently can't read today 😄